hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFileSystem.java

```diff
@@ -802,6 +802,7 @@ protected void deleteRootDir() throws IOException, InterruptedException {
       return;
     }
     deleteRootRecursively(fileStatuses);
+    Thread.sleep(500);
```

Comment:
Can we use [GenericTestUtils#waitFor](https://github.com/apache/ozone/blob/387c53781aae03f944906a0add58f51250d36687/hadoop-hdds/test-utils/src/main/java/org/apache/ozone/test/GenericTestUtils.java#L214) instead of `Thread.sleep()`?

Reply:
> Can we use [GenericTestUtils#waitFor](https://github.com/apache/ozone/blob/387c53781aae03f944906a0add58f51250d36687/hadoop-hdds/test-utils/src/main/java/org/apache/ozone/test/GenericTestUtils.java#L214) instead of `Thread.sleep()`?

Thank you @hemantk-12 for review. I have handled the comment.

Reply:
> Can we use [GenericTestUtils#waitFor](https://github.com/apache/ozone/blob/387c53781aae03f944906a0add58f51250d36687/hadoop-hdds/test-utils/src/main/java/org/apache/ozone/test/GenericTestUtils.java#L214) instead of `Thread.sleep()`?

@hemantk-12 thanks for the review, I tried using GenericTestUtils, however this in actual run throwing always TimeOutException and not working in this case.

Reply:
I'm surprise that `Thread.sleep()` works fine but [GenericTestUtils#waitFor](https://github.com/apache/ozone/blob/387c53781aae03f944906a0add58f51250d36687/hadoop-hdds/test-utils/src/main/java/org/apache/ozone/test/GenericTestUtils.java#L214) doesn't. Just curious what was lambda used?

I feel this should work if it is just about cleaning dir.
```
    GenericTestUtils.waitFor(() -> {
      try {
        FileStatus[] fileStatus = fs.listStatus(ROOT);
        return fileStatus != null && fileStatus.length == 0;
      } catch (IOException e) {
        return false;
      }
    }, 100, 500);
    ```

Reply:
Thanks @hemantk-12 for review. I have handled as per your suggestion.